### PR TITLE
Fixes for lmg

### PR DESCRIPTION
**Pull request by lmgDigital2023**

### CODE REVIEW

File: android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java [hunk 1]
```diff
@@ -91,6 +92,7 @@ public void changeIcon(String iconName, Promise promise) {
         this.componentClass = activeClass;
         activity.getApplication().registerActivityLifecycleCallbacks(this);
         iconChanged = true;
+        this.completeIconChange();
```
Comment (abhishekshori1) [inline android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java:96]: Please add a comment why is this required

File: ios/ChangeIcon.mm [hunk 2]
```diff
@@ -18,7 +18,7 @@ + (BOOL)requiresMainQueueSetup {
     });
 }
 
-RCT_REMAP_METHOD(changeIcon, iconName:(NSString *)iconName resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+RCT_REMAP_METHOD(changeIcon, iconName:(NSString *)iconName newPackageName:(NSString *)newPackageName resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```
Comment (abhishekshori1) [inline ios/ChangeIcon.mm:22]: Add a comment here as well.

File: android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java (hunk 1 above)
Comment (lmgDigital2023) [reply android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java:96]: @abhishekshori1 Addressed.

File: ios/ChangeIcon.mm (hunk 2 above)
Comment (lmgDigital2023) [reply ios/ChangeIcon.mm:22]: @abhishekshori1 Addressed.
